standalone/src/models/shipping-method/shipping-rate/shipping-rate-price-tier/cart-value/builders.spec.ts

```diff
@@ -0,0 +1,71 @@
+import type {
+  TCartValuePriceTierRest,
+  TCartValuePriceTierGraphql,
+} from './types';
+import {
+  CartValuePriceTierDraftRest,
+  CartValuePriceTierGraphql,
+} from './index';
+
+function validateRestModel(model: TCartValuePriceTierRest) {
+  expect(model).toEqual(
+    expect.objectContaining({
+      type: 'CartValue',
+      minimumCentAmount: expect.any(Number),
+      price: expect.objectContaining({
+        centAmount: expect.any(Number),
+        currencyCode: expect.any(String),
+      }),
+      isMatching: expect.any(Boolean),
+    })
+  );
+}
+
+function validateGraphqlModel(model: TCartValuePriceTierGraphql) {
```

Comment:
suggestion: We could consolidate just one validation method and check the `__typename` only in the Graphql tests.